src/platform/site-wide/announcements/containers/Announcement.jsx

```diff
@@ -55,11 +78,10 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = {
-  initDismissedAnnouncements,
   dismissAnnouncement,
+  initDismissedAnnouncements,
 };
 
-export { Announcement };
 export default connect(
   mapStateToProps,
   mapDispatchToProps,
```

Comment:
I see the addition of `propTypes` but otherwise I'm not sure I understand the changes in this file. Is it just a cleanup?

Reply:
Yupp! Just was reading through it to understand what it was doing and cleaned up at the same time 🙂